app/views/helpers/FullResultsPageHelper.scala

```diff
@@ -176,51 +184,50 @@ object FullResultsPageHelper extends ViewHelper {
-            TableRow(content = Text(messages("fullResultsPage.financialYear.adjustUpperLimit"))),
-            TableRow(content =
-              Text(s"$upperLimit $upperLimitMsg × ($daysString ÷ $daysInYear $daysMsg) ÷ $pointOneCompaniesCalcText")
-            ),
-            TableRow(content = Text(CurrencyUtils.format(adjustedUpperLimit)))
-          ),
-          Seq(
-            boldRow("2"),
-            TableRow(content = Text(messages("fullResultsPage.financialYear.taxableProfit"))),
-            TableRow(content = Text(s"${CurrencyUtils.format(taxableProfit)} × ($daysString ÷ $daysInYear $daysMsg)")),
-            TableRow(content = Text(CurrencyUtils.format(marginalRate.adjustedProfit)))
-          ),
-          Seq(
-            boldRow("3"),
-            TableRow(content = Text(messages("fullResultsPage.financialYear.taxableProfitDistributions"))),
-            TableRow(content =
-              Text(
-                s"${CurrencyUtils.format(marginalRate.adjustedProfit)} + ${CurrencyUtils
-                    .format(distributions)} × ($daysString ÷ $daysInYear $daysMsg)"
-              )
-            ),
-            TableRow(content = Text(CurrencyUtils.format(taxableProfitIncludingDistributions)))
-          ),
-          Seq(
-            boldRow("4"),
-            TableRow(content = Text(messages("fullResultsPage.financialYear.marginalReliefFraction"))),
-            TableRow(content = Text(messages("fullResultsPage.financialYear.marginalReliefFraction.description"))),
-            TableRow(content = Text(fraction))
-          ),
-          Seq(
-            boldRow("5"),
-            TableRow(content = Text(messages("fullResultsPage.financialYear.fullCalculation"))),
-            TableRow(content = Text(s"""(${CurrencyUtils.format(adjustedUpperLimit)} - ${CurrencyUtils.format(
-                taxableProfitIncludingDistributions
-              )}) × (${CurrencyUtils.format(
-                marginalRate.adjustedProfit
-              )} ÷ ${CurrencyUtils.format(taxableProfitIncludingDistributions)}) × ($fraction)""")),
-            TableRow(content = Text(CurrencyUtils.format(ResultsPageHelper.marginalRelief(marginalRate))))
+    val isUpperLowerLimit = taxableProfitIncludingDistributions >= marginalRate.adjustedLowerThreshold
+
+    val firstThreeSteps = Seq(
+      Seq(
+        boldRow("1"),
+        TableRow(content =
+          Text(
+            messages(
+              if (isUpperLowerLimit) "fullResultsPage.financialYear.adjustedUpperLimit"
+              else "fullResultsPage.financialYear.adjustedLowerLimit"
+            )
+          )
+        ),
+        TableRow(content =
+          Text(
+            s"${if (isUpperLowerLimit) upperLimitText else lowerLimitText} × ($daysString ÷ $daysInYear $daysMsg) ÷ $pointOneCompaniesCalcText"
           )
         ),
+        TableRow(content =
+          Text(CurrencyUtils.format(if (isUpperLowerLimit) adjustedUpperLimit else marginalRate.adjustedLowerThreshold))
```

Comment:
Can we use consistent language here? We have always used threshold, please replace limit with threshold